whois-nrtm4-client/src/main/java/net/ripe/db/nrtm4/client/importer/SnapshotMirrorImporter.java

```diff
@@ -62,32 +86,51 @@ public void doImport(final String source,
             return;
         }
 
+        final AtomicInteger snapshotVersion = new AtomicInteger(0);
         final AtomicInteger processedCount = new AtomicInteger(0);
+
+        persisSnapshot(source, payload, sessionId, snapshotVersion, processedCount);
+        persistSnapshotVersion(source, snapshotVersion.get(), sessionId);
+
+        stopwatch.stop();
+        LOGGER.info("Loading snapshot file took {} for source {} and added {} records", stopwatch.elapsed().toMillis(), source, processedCount.get());
```

Comment:
No need to convert snapshot to milliseconds, the `toString()` will always print something readable.